app/main.py

```diff
@@ -2,0 +1,24 @@
+from app.cafe import Cafe
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except NotVaccinatedError:
+            return "All friends should be vaccinated"
+        except OutdatedVaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks_to_buy += 1
+        except KeyError:
+            return "All friends should be vaccinated"
```

Comment:
The message 'All friends should be vaccinated' is used for `KeyError`, which might not be accurate. Consider checking which key is missing and providing a more specific message.